packages/pyright-internal/src/baseline.ts

```diff
@@ -273,7 +273,8 @@ export class BaselineHandler {
             files: {},
         };
         for (const fileWithDiagnostics of filesWithDiagnostics) {
-            const filePath = this.configOptions.projectRoot.getRelativePath(fileWithDiagnostics.fileUri)!.toString();
+            const filePath = this.configOptions.projectRoot.getRelativePath(fileWithDiagnostics.fileUri)?.toString();
+            if (filePath === undefined) continue;
```

Comment:
did you encounter a crash here that prompted this change? if so are you able to reproduce it?

Reply:
Yes I crashed here, it seems to happen when some file doesn't have inspections but is nonetheless added into the list of files that have inspections

Reply:
Ok so actually

I think this happened because I have my test files in a different folder from the Pyright's running directory when using the debugger, which is the root `basedpyright` directory

Reply:
fixed in #982 so this change can now be reverted